### ISSUE TITLE
jsk su and sudo not found.

### ISSUE THREAD

**Issue by LeoCx1000** [Gorialis/jishaku#120]
## Summary

the commands `jsk su` and `jsk sudo` don't seem to be found.

## Reproduction steps

1) load the latest version of discord.py and jishaku
2) do `db.jsk su @user help` or `db.jsk sudo help`
3) do `db.help jsk su` or `db.help jsk sudo` (subclassed help cmd)

## Expected results

1) *shows help command as if executed by `@user`* / *runs command ignoring checks*
2) *shows information about jishaku su/sudo command*

## Actual results

2) jishaku command raised an error:
    Too many arguments passed to jishaku

3) Command "jishaku" has no subcommand named su/sudo

## Checklist

- [x] I have updated discord.py and jishaku to the latest available versions and have confirmed that this issue is still present
- [x] I have searched the open issues for duplicates
- [x] I have shown the entire traceback, if possible
- [x] I have removed my token from display, if visible
- [x] not a self-bot

## System information

```
- Python v3.8.10-final
- discord.py v1.7.3-final
- aiohttp v3.7.4.post0
- system info: Linux 5.8.0-1038-aws #40~20.04.1-Ubuntu SMP Thu Jun 17 13:25:28 UTC 2021
```

```
Name: jishaku
Version: 2.2.0
Summary: A discord.py extension including useful tools for bot development and debugging.
Home-page: https://github.com/Gorialis/jishaku
Author: Devon (Gorialis) R
Author-email:
License: MIT
Location: /home/ubuntu/.local/lib/python3.8/site-packages
Requires: braceexpand, click, import-expression, discord.py
Required-by:
```

**Comment by Gorialis** [comment]
Duplicate of #118
